boards/arm/stm32h735g_disco/stm32h735g_disco_defconfig

```diff
@@ -26,3 +26,9 @@ CONFIG_PINCTRL=y
 
 # select power supply
 CONFIG_POWER_SUPPLY_DIRECT_SMPS=y
+
+# use LPTIM as clock instead of SYSTICK
+CONFIG_CORTEX_M_SYSTICK=n
+
+# activate power management
+CONFIG_PM=y
```

Comment:
CONFIG_PM should not be enabled as part of default board configuration.
This configuartion is application responsibility and should be done in samples.

Reply:
done